pkg/certificate/providers/vault/types.go

```diff
@@ -2,36 +2,16 @@
 package vault
 
 import (
-	"sync"
-	"time"
-
 	"github.com/hashicorp/vault/api"
-
-	"github.com/openservicemesh/osm/pkg/certificate"
-	"github.com/openservicemesh/osm/pkg/configurator"
-	"github.com/openservicemesh/osm/pkg/messaging"
 )
 
 // CertManager implements certificate.Manager and contains a Hashi Vault client instance.
 type CertManager struct {
-	// The Certificate Authority root certificate to be used by this certificate manager
-	ca *certificate.Certificate
-
-	// Cache for all the certificates issued
-	// Types: map[certificate.CommonName]*certificate.Certificate
-	cache sync.Map
-
 	// Hashicorp Vault client
 	client *api.Client
 
 	// The Vault role configured for OSM and passed as a CLI.
 	role vaultRole
-
-	cfg configurator.Configurator
-
-	serviceCertValidityDuration time.Duration
-
-	msgBroker *messaging.Broker
 }
 
 type vaultRole string
```

Comment:
the benefit of aliasing types is typically to reduce the allowed set of values. ie:

```
type vaultRole string // unexported, so needs to be specified

var (
   defaultRole vaultRole = "default-role"
)
```

Since we're not doing that, let's just go ahead and get rid of these and call them strings.